src/Documentation/pages.ts

```diff
@@ -61,9 +61,8 @@ import file58 from "!!raw-loader!./doc/programming/hackingalgorithms.md";
 import file59 from "!!raw-loader!./doc/programming/learn.md";
 import file60 from "!!raw-loader!./doc/programming/remote_api.md";
 
-interface Document {
-  default: string;
-}
+import type { Document } from "./root.js";
```

Comment:
No particular reason, other than unfamiliarity with javascript/typescript.

Reply:
It works because typescript will look for `.ts` when asked for `.ts`. I think this partly because typescript tries to not change the dynamic behaviour of programs, so when not bundling, the import would eventually refer to a `.js` file.